internal/mode/static/telemetry/collector_test.go

```diff
@@ -298,6 +299,11 @@ var _ = Describe("Collector", Ordered, func() {
 						client.ObjectKeyFromObject(svc2):   {},
 						client.ObjectKeyFromObject(nilsvc): {},
 					},
+					BackendTLSPolicies: map[types.NamespacedName]*graph.BackendTLSPolicy{
+						{Namespace: "test", Name: "bgPolicy-1"}: {},
```

Comment:
nit: what is `bg` short for in `bgPolicy`?

Reply:
aah typo. means `backendTLS` , let me rename it properly